src/SSHDebugPS/Docker/DockerConnection.cs

```diff
@@ -243,10 +245,14 @@ private ICommandRunner GetCommandRunner(DockerContainerTransportSettings setting
                 return new RemoteCommandRunner(settings, OuterConnection);
         }
 
+        private static readonly string IncorrectContainerType = "IncorrectContainerType";
```

Comment:
if you are adding this for telemetry, please follow @WardenGnaw 's pattern and add it to the telemtry file and not in line with the code. Telemetry events and properties need to be prefaced correctly.